bool store_carry_forwarding
bool channel_offloading
uint8 id # only lower six bits are used


uint8 ID_DCC_DP0 = 0
uint8 ID_DCC_DP1 = 1
uint8 ID_DCC_DP2 = 2
uint8 ID_DCC_DP3 = 3
